app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `Person.people` dictionary is used to store instances of `Person` by their name. This approach assumes that each person has a unique name. If names are not unique, this will lead to overwriting existing entries. Consider using a unique identifier for each person, such as an ID.